tests/functional/test_plugins.py

```diff
@@ -995,6 +1014,16 @@ def test_syft_binary(copy_files: list[Path], chalk_copy: Chalk, use_docker: bool
     chalk_mark = ChalkMark.from_binary(bin_path)
     assert chalk_mark.contains(sbom_data)
 
+    insert2 = chalk.insert(
+        bin_path,
+        env={
+            "EXTERNAL_TOOL_USE_DOCKER": str(use_docker),
+            "RUNNER_TEMP": str(tmp_data_dir),
+            "CI": "true",
+        },
+    )
+    insert2.mark.has(SBOM=insert.mark["SBOM"])
+
```

Comment:
is this testing that the second insert has the _same exact_ SBOM mark as the first insert?

Reply:
yes. sbom contains a timestamp so if they are both equivalent it must of been cached in-between inserts